backend/requirements.txt

```diff
@@ -0,0 +1,107 @@
+apt-clone==0.2.1
+apturl==0.5.2
+astroid==2.4.1
+attrs==19.3.0
+beautifulsoup4==4.6.0
+Brlapi==0.6.6
+certifi==2020.4.5.2
+cffi==1.14.0
+chardet==3.0.4
+cli-helpers==2.0.1
+click==7.1.2
+command-not-found==0.3
+configobj==5.0.6
+coverage==5.1
+cryptography==2.9.2
```

Comment:
yeah these are probably nested dependencies; by doing `pip freeze` it initializes the requirements.txt, with all the current requirements including nested dependencies...can you try on your end to generate requirements and see what it comes up with?

Reply:
I tried pip install -r and it runs into an error. I think it would be best to initialize a Conda env, and manually putting all the things in the requirements. using pip freeze generates lots of nested dependencies, which may not be available in the PyPI repositories.

Reply:
Ok, looks like you already fixed requirements.txt